mne_bids_pipeline/_docs.py

```diff
@@ -0,0 +1,261 @@
+                                continue
+                            self._add_step_option(step, keyword.value.attr)
+                    # Also look for root-level conditionals like use_maxwell_filter
+                    # or spatial_filter
+                    for cond in ast.iter_child_nodes(func):
+                        # is a conditional
+                        if not isinstance(cond, ast.If):
+                            continue
+                        # has a return statement
+                        if not any(isinstance(c, ast.Return) for c in ast.walk(cond)):
+                            continue
+                        # look at all attributes in the conditional
+                        for attr in ast.walk(cond.test):
+                            if not isinstance(attr, ast.Attribute):
+                                continue
+                            if attr.value.id != "config":
+                                continue
+                            self._add_step_option(step, attr.attr)
+                # Now look at get_config* functions
+                if not func.name.startswith("get_config"):
+                    continue
+                found = True
+                for call in ast.walk(func):
+                    if not isinstance(call, ast.Call):
+                        continue
+                    if call.func.id != "SimpleNamespace":
+                        continue
+                    break
+                else:
+                    raise RuntimeError(f"Could not find SimpleNamespace in {func}")
+                assert call.args == []
+                for keyword in call.keywords:
+                    if isinstance(keyword.value, ast.Call):
+                        key = keyword.value.func.id
+                        if key in _MANUAL_KWS:
+                            for option in _MANUAL_KWS[key]:
+                                self._add_step_option(step, option)
+                            continue
+                        if keyword.value.func.id == "_sanitize_callable":
+                            assert len(keyword.value.args) == 1
+                            assert isinstance(keyword.value.args[0], ast.Attribute)
+                            assert keyword.value.args[0].value.id == "config"
+                            self._add_step_option(step, keyword.value.args[0].attr)
+                            continue
+                        if key not in (
+                            "_bids_kwargs",
+                            "_import_data_kwargs",
+                            "get_runs",
+                            "get_subjects",
+                            "get_sessions",
+                        ):
+                            raise RuntimeError(
+                                f"{where} cannot handle call {keyword.value.func.id=} "
+                                f"for {key}"
+                            )
+                        # Get the source and regex for config values
```

Comment:
This file is copy-paste then refactored to have constants at the top, plus added regex for finding `config.` stuff in functions like `_bids_kwargs` and `get_runs` etc.